eng/Versions.props

```diff
@@ -17,6 +17,7 @@
   <PropertyGroup>
     <VersionPrefix>16.7.0</VersionPrefix>
     <AssemblyVersion>15.1.0.0</AssemblyVersion>
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Delete this and amend it into the merge commit (and make a note in the commit message that you did so).